core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -293,14 +294,14 @@ public T enableFullStreamDecompression() {
    * <p>This method is advisory, and implementations may decide to not enforce this.  Currently,
    * the only known transport to not enforce this is {@code InProcessTransport}.
    *
-   * @param max the maximum number of bytes a single message can be.
-   * @throws IllegalArgumentException if max is negative.
+   * @param bytes the maximum number of bytes a single message can be.
    * @return this
+   * @throws IllegalArgumentException if bytes is negative.
    * @since 1.1.0
    */
-  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/2307")
-  public T maxInboundMessageSize(int max) {
-    // intentional nop
+  public T maxInboundMessageSize(int bytes) {
+    // intentional noop rather than throw, this method is only advisory.
```

Comment:
I think you need to have both methods for a short time.  We typically have one release with deprecation before removing it.

Also, this makes it easier to make internal users update before removal.

Reply:
No methods were removed in this PR.

Reply:
Yes, I just renamed `max` to `bytes` to be consistent.